### DISCUSSION TITLE
How to lock chromium cask at version 126?

### DISCUSSION THREAD

**Discussion by unikitty37** [Homebrew/discussions#5544] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: 1e560120a65be3b3c7b8a42083d54f795e1d0ec4
Last commit: 30 hours ago
Core tap JSON: 06 Aug 13:15 UTC
Core cask tap JSON: 06 Aug 13:15 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_BROWSER: /usr/bin/open
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: set
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.45.2 => /opt/homebrew/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1

Warning: You have the following deprecated, official taps tapped:
  Homebrew/homebrew-cask-versions
  Homebrew/homebrew-cask-fonts
Untap them with `brew untap`.
```

### Description of issue

Chromium 127 starts the removal of Manifest v2, and all the useful things that enables (like uBlock Origin) will be hidden behind a sea of nags before being blocked completely.

It would be really useful to be able to type `brew install --cask chromium@126` and have Chromium 126 installed without getting updated to the broken-by-design version.

### Relevant casks

chromium

**Comment by Bo98** [comment]
For security reasons, we don't have a `chromium@126`. But there may be forks we package that achieve something similar like `eloston-chromium` (aka ungoogled-chromium) that may continue to support Manifest v2 while keeping up-to-date with security patches: https://github.com/ungoogled-software/ungoogled-chromium/issues/662
